ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -70,8 +70,8 @@ The following data model keys have been removed from `eos_cli_config_gen` in v5.
 | old key 4.2(management_api_gnmi) | new key(TODO) |
 | management_security.entropy_source | management_security.entropy_sources |
 | old key 6(name_server) | new key(TODO) |
-| old key 7.1(port_channel_interfaces) | new key(TODO) |
-| old key 7.2(port_channel_interfaces) | new key(TODO) |
+| port_channel_interfaces.esi | port_channel_interfaces.evpn_ethernet_segment.identifier |
```

Comment:
| port_channel_interfaces[].esi | port_channel_interfaces[].evpn_ethernet_segment.identifier |

Can we add like this.

Reply:
yes

Reply:
But you also need to add rt -> route_target

Reply:
updated